llarp/service/info.hpp

```diff
@@ -101,11 +104,8 @@ namespace llarp
       bool
       DecodeKey(const llarp_buffer_t& key, llarp_buffer_t* buf);
     };
-
-    inline std::ostream&
-    operator<<(std::ostream& out, const ServiceInfo& i)
-    {
-      return i.print(out, -1, -1);
-    }
   }  // namespace service
 }  // namespace llarp
+
+template <>
+constexpr inline bool llarp::IsToStringFormattable<llarp::service::ServiceInfo> = true;
```

Comment:
wait... so does this template specialization belong inside or outside the llarp namespace?